src/ontology/mondo-edit.obo

```diff
@@ -29306,6 +29306,8 @@ xref: SCTID:427795000 {source="MONDO:equivalentTo", source="DOID:14457"}
 xref: SCTID:9060000 {source="DOID:14457"}
 xref: UMLS:C0302363 {source="MONDO:notFoundInDiseaseSubset", source="DOID:14457"}
 is_a: MONDO:0005683 {source="DOID:14457", source="linkedlifedata"} ! brucellosis
+intersection_of: MONDO:0005683 ! brucellosis
```

Comment:
This is good, but in future consider modifying the text def so that they align, especially if the source def is something like mesh.